setup.py

```diff
@@ -51,7 +51,7 @@
     'jsonpickle>=0.9.6',
     'marshmallow>2.13.0,<=2.20.1',
     'packaging>=18.0',
-    'reana-commons[kubernetes]>=0.6.0.dev20191128,<0.7.0',
+    'reana-commons[kubernetes]>=0.6.0.dev20191129,<0.7.0',
```

Comment:
This kind of version upgrade is good to do *after* the reana-commons is released on PyPI... Otherwise one cannot test locally without amending it...